ROADMAP.md

```diff
@@ -0,0 +1,185 @@
+- allow compilation of helpers (write in esnext?) https://github.com/babel/babel/issues/5876
+
+## Repurpose `babel` / create `babel-init`
+
+> We can re-use the `babel` package for a more simplified *0 config™* version
+
+Not sure what this looks like but had this idea for a really long time and didn't really get anywhere - the cli version could just be https://github.com/developit/microbundle for libs? Maybe webpack/parcel would have it covered for apps?
+
+---
+
+## General/Infra
+
+### Run Babel against Spec tests (test262, TS, Flow, JSX)
+
+Better set of tests for stability/spec compliancy.
+
+- The parser (babylon) already has a whitelist of test262 tests
+- Need to do the same for the transforms.
+
+### Smoke Tests
+
+- Babel itself https://github.com/babel/babel/issues/6134
+- Important community plugins (`babel-plugin-instanbul`, css-in-js)
+- `babel-core` integrations (wrappers like `babel-loader`, test frameworks, etc)
+- Libraries (`(p)react`, `redux`, `vue`)
+- Tools that use individual packages (`debugger.html`, `prepack`)
+- OSS Apps (`nylas-mail`)
+
+### Better Debuggability
+
+- Query config for data for other tools
+- `babel --settings`
+- Validate config better
+- Create/expand on new tools like https://github.com/boopathi/babel-time-travel
+
+### Speed
+  
+> Already working with v8 via https://github.com/v8/web-tooling-benchmark, but can add other representative workloads: jsx/flow/ts/es6+.
+
+Can run these benchmarks for perf PRs, should track some over time.
+
+### Website Rehaul
+
+- Use a blog framework like Gatsby/Docusaurus
+- Versioned docs pages: currently we don't have an easy way to show both the documentation for v6, v7, and beyond.
+- Translatable docs
+
+#### Expanded Docs
+
+- Real documentation on APIs
+- Up to date babel-handbook/merge into rehauled website
+- Continue our [videos page](https://babeljs.io/docs/community/videos/)
+- Link to common errors pages
+
+#### Better REPL
+
+- Dropdown examples/examples of syntax from github?
+- Import any package from npm (can give test examples for 3rd party plugins, debugging issues)
+- Run any plugin from npm
+- Create a plugin from the repl (can we merge it with ASTExplorer/codesandbox?), even publish, run from URL?
+- Import/Export a `.babelrc` file
+- Combine ^ with the ability to run the version of Babel in a PR/master.
+- Use plugin's tests in the repo as "examples" for docs.
+
+### `babel-bot`
+
+> [#43](https://github.com/babel/babel-bot/issues/43) Rewrite it with [probot](https://github.com/probot/probot)
+
+A bot is really useful to do github/maintainer activities automatically.
+
+We haven't updated this in a long time due to Andrew being busy and not setting up the automatic infra on AWS. Switching will make updating actually real so we can add some new features which would save some headache.
+
+References: https://github.com/eslint/eslint-github-bot, https://github.com/open-bot/open-bot
```

Comment:
[Peril](https://github.com/danger/peril) is also an option we've deprecated a bunch of CocoaPods/Fastlane micro-servers to just a few lines of JS